app/helpers/parsers_helper.rb

```diff
@@ -1,49 +1,82 @@
-# The majority of The Supplejack Manager code is Crown copyright (C) 2014, New Zealand Government,
-# and is licensed under the GNU General Public License, version 3. Some components are 
-# third party components that are licensed under the MIT license or otherwise publicly available. 
-# See https://github.com/DigitalNZ/supplejack_manager for details. 
-# 
-# Supplejack was created by DigitalNZ at the National Library of NZ and the Department of Internal Affairs. 
+# The majority of The Supplejack Manager code is Crown copyright (C) 2014,
+# New Zealand Government,
+# and is licensed under the GNU General Public License, version 3.
+# Some components are third party components that are licensed under
+# the MIT license or otherwise publicly available.
+# See https://github.com/DigitalNZ/supplejack_manager for details.
+#
+# Supplejack was created by DigitalNZ at the National Library of NZ and
+# the Department of Internal Affairs.
 # http://digitalnz.org/supplejack
 
+# Helpers for Parsers
 module ParsersHelper
-
-  def link_to_next(parser_id, index, environment, review, html_options={})
-    link_to "next >", preview_path(parser_id, index: index.to_i+1, environment: environment, review: review), html_options
+  # Returns the next link for preview view
+  #
+  # @author Federico Gonzalez
+  # @last_modified Eddie
+  # @param parser_id [String] the id fo the parser
+  # @param index [String] the index of the preview page
+  # @param environment [String] the environment set to the parser
+  # @param review [String]
+  # @param html_options [String] optional options
+  #
+  # @return [A Tag] of the next preview page
+  def link_to_next(parser_id, index, environment, review, html_options = {})
+    path = parser_previewers_path(parser_id,
+                                  index: index.to_i + 1,
```

Comment:
Put a comment explaining why index has 1 added to it, I can guess at a few reasons but I can't be sure which is the correct one.